spec/lib/diaspora/exporter_spec.rb

```diff
@@ -10,6 +17,14 @@
       user,
       %i[generic_user_data activity status_messages_flavours work_aspect]
     )
+
+    # verifications of data generation (protect us from possible false positives in case of poor data preset)
```

Comment:
We have a `spec/misc_spec.rb` with "tests for tests" now. I can put it to this file or create something else (another file or folder)?